cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -296,10 +296,10 @@ def upload_splice_junctions_bed_to_scout(self, dry_run: bool, case_id: str) -> N
         status_db: Store = self.status_db
         rna_case: Family = status_db.get_case_by_internal_id(internal_id=case_id)
 
-        rna_dna_sample_case_map: Dict[str, Dict[str, list]] = self.create_rna_dna_sample_case_map(
-            rna_case=rna_case
-        )
-        for rna_sample_id in rna_dna_sample_case_map:
+        rna_dna_bundles: List[RnaDnaBundle] = self.create_rna_dna_bundles(rna_case=rna_case)
```

Comment:
Skip named argument if you think it is more readable. Does not really matter here since it does not introduce line breaks, skipping it is a little less verbose